paper/joss_preview/paper.Rmd

```diff
@@ -0,0 +1,165 @@
+---
+title: 'bbousuite: A set of R packages to facilitate analysis of Boreal Caribou survival and recruitment data'
+authors:
+- name: Seb Dalgarno
+  orcid: 0000-0002-3658-4517
+  affiliation: 1
+- name: John Boulanger
+  orcid: 0000-0001-8222-1445
+  affiliation: 2
+- name: Ayla Pearson
+  orcid: 0000-0001-7388-1222
+  corresponding: true
+  affiliation: 1
+- name: Joe Thorley
+  orcid: 0000-0002-7683-4592
+  affiliation: 1
+- name: Troy Hegel  
+  affiliation: 3
+- name: Barry Nobert
+  affiliation: 3
+- name: Dave Hervieux
+  affiliation: 3
+affiliations:
+- index: 1
+  name: Poisson Consulting, Canada
+- index: 2
+  name: Integrated Ecological Research, Canada
+- index: 3
+  name: Province of Alberta, Canada
+date: 6 May 2024
+bibliography: paper.bib
+output: 
+  rticles::joss_article: default
+csl: apa.csl
+journal: JOSS
+tags:
+  - boreal caribou
+  - Bayesian
+  - maximum likelihood
+  - survival
+  - recruitment
+  - population growth
+---
+
+# Summary
+
+Although many animal populations are managed based on their abundance [@williams_analysis_2002], reliable abundance estimates can be impractical to obtain for mobile herding populations with large ranges such as boreal caribou (*Rangifer tarandus caribou*).
+Consequently, such populations are commonly managed based on their population growth rate ($\lambda$), which can be estimated from the population recruitment and adult female survival rates [@hatter_moose_1991]. 
+
+The `bbou` suite is a set of R packages [@r_core_team_2023] to facilitate simulation and analysis of boreal caribou survival and recruitment data to estimate population growth.  
+
+It consists of:  
+- [`bboudata`](https://poissonconsulting.github.io/bboudata/), a collection of simulated and anonymized empirical survival and recruitment datasets.  
+- [`bboutools`](https://poissonconsulting.github.io/bboutools/), tools to estimate population growth in a Bayesian or Maximum Likelihood (ML) framework, implemented under the hood using the `Nimble` R package [@de_valpine_programming_2017].  
+- [`bboushiny`](https://poissonconsulting.github.io/bboushiny/), a Graphical User Interface (GUI) to `bboutools` (\autoref{fig:bboushiny}).    
```

Comment:
The autoref is not working when I render to html locally? 

<img width="954" alt="Screenshot 2024-09-05 at 07 14 27" src="https://github.com/user-attachments/assets/cfae9ddf-9496-468e-9fa3-617c995f9cba">

Reply:
yes I'm not worried about this as it works when render as pdf and joss article